### DISCUSSION TITLE
A dontSave() method for form components

### DISCUSSION THREAD

**Discussion by edgrosvenor** [filamentphp/filament#302] [Help] [answered]
There might already be an easy way to do this, but I haven't found it yet. I have created a custom form component because I want certain data displayed to me while editing a record, but that component doesn't actually correspond to a column in the database. Unfortunately if I try to use it, it tries to save data in the column that corresponds to the name of the component. So what I'd love is something like this...

`VideoPlayer::make('embed')->label('Preview Video')->dontSave()` 

Basically this would allow me to create dummy components that sit in the form but don't actually impact the record.

**Comment by zhanang19** [comment, marked as answer]
How about disabling the input? Any disabled input will not submitted

**Reply by ryangjchandler** [reply]
> How about disabling the input? Any disabled input will not submitted

I'm not sure if Livewire respects `disabled` inputs with `wire:model` or not.

**Reply by edgrosvenor** [reply]
That didn't work. There's not actually an input in there at all. It's actually just a video player. There's not any way I can see, though, to add content to a form that's not a form component.

**Reply by ryangjchandler** [reply]
There's a `View` field type that you could use.

**Reply by edgrosvenor** [reply]
Doh! If I'd read just one section past the "Create Custom Component" block of the docs I'd have found that. Thanks! Works perfectly!
